README.md

```diff
@@ -1,2 +1,34 @@
 # storcycle_api
-StorCycle API Examples
+###StorCycle API Examples
```

Comment:
Need a space between your # and the title, same as below.  And you should use a progression of #, so this one would have 2, not 3, and the python one below should have 3, not 4.